cg/meta/workflow/nf_analysis.py

```diff
@@ -649,11 +648,13 @@ def get_deduplicated_metrics(metrics: list[MetricsBase]) -> list[MetricsBase]:
                 deduplicated_metrics.append(metric)
         return deduplicated_metrics
 
+    def get_multiqc_data_json(self, case_id: str) -> MultiqcDataJson:
+        """Return a MultiqcDataJson object."""
```

Comment:
I think this docstring just duplicates the return type written a line above.

Reply:
lets remove the docstring @peterpru